components/gallery/GalleryFileFormat.js

```diff
@@ -0,0 +1,35 @@
+const galleryFileFormats = [
+    { extension: ".glb", encoding: "model/gltf-binary", type: "3DModel" },
```

Comment:
To address https://github.com/Geo-Web-Project/cadastre/issues/89, I believe these should just be listed in the following order:

{ extension: ".gif", encoding: "image/gif", type: "ImageObject" },
{ extension: ".glb", encoding: "model/gltf-binary", type: "3DModel" },
{ extension: ".jpeg", encoding: "image/jpeg", type: "ImageObject" },
{ extension: ".mp3", encoding: "audio/mpeg", type: "AudioObject" },
{ extension: ".mp4", encoding: "audio/mp4", type: "AudioObject" },
{ extension: ".mp4", encoding: "video/mp4", type: "VideoObject" },
{ extension: ".mpeg", encoding: "video/mpeg", type: "VideoObject" },
{ extension: ".png", encoding: "image/png", type: "ImageObject" },
{ extension: ".svg", encoding: "image/svg+xml", type: "ImageObject" },
{ extension: ".usdz", encoding: "model/vnd.usdz+zip", type: "3DModel" }